core/src/it/scala/org/ensime/core/RichPresentationCompilerSpec.scala

```diff
@@ -598,7 +598,7 @@ trait RichPresentationCompilerTestUtils {
   }
 
   def srcFile(proj: EnsimeConfig, name: String, content: String, write: Boolean = false, encoding: String = "UTF-8"): BatchSourceFile = {
-    val src = proj.projects.head.sources.head / name
+    val src = proj.projects.head.sources.head.file.resolve(name).toFile
```

Comment:
you also get some methods on the `Rawfile` by importing `org.ensime.util.ensimefile._`